src/Views/AbstractView.vala

```diff
@@ -18,60 +18,74 @@
 *
 */
 
-public abstract class AbstractView : Gtk.Grid {
-    protected Gtk.ButtonBox button_box;
+public abstract class AbstractView : Gtk.Box {
+    protected Gtk.Box button_box;
     protected Gtk.Grid content_area;
     protected Gtk.Image badge;
     protected Gtk.Label primary_label;
     protected Gtk.Label secondary_label;
 
     construct {
-        var image = new Gtk.Image.from_icon_name ("io.elementary.sideload", Gtk.IconSize.DIALOG);
+        var image = new Gtk.Image.from_icon_name ("io.elementary.sideload") {
+            pixel_size = 48
+        };
         image.valign = Gtk.Align.START;
 
         badge = new Gtk.Image ();
         badge.pixel_size = 24;
         badge.halign = badge.valign = Gtk.Align.END;
 
-        var overlay = new Gtk.Overlay ();
-        overlay.valign = Gtk.Align.START;
-        overlay.add (image);
+        var overlay = new Gtk.Overlay () {
+            child = image,
+            valign = Gtk.Align.START
+        };
         overlay.add_overlay (badge);
 
-        primary_label = new Gtk.Label (null);
-        primary_label.hexpand = true;
-        primary_label.max_width_chars = 50;
+        primary_label = new Gtk.Label (null) {
+            max_width_chars = 1,
+            xalign = 0
+        };
+
         primary_label.selectable = true;
         primary_label.wrap = true;
-        primary_label.xalign = 0;
-        primary_label.get_style_context ().add_class (Granite.STYLE_CLASS_PRIMARY_LABEL);
+        primary_label.add_css_class (Granite.STYLE_CLASS_TITLE_LABEL);
 
-        secondary_label = new Gtk.Label (null);
+        secondary_label = new Gtk.Label (null) {
+            max_width_chars = 50,
+            width_chars = 50,
+            xalign = 0
+        };
```

Comment:
Any reason why other properties not initialised in constructor?

Reply:
No, I don't have a good reason. I was probably just trying to keep diff size small. Updated!